pepper-apis/studybuilder-cli/studies/pancan/subs.conf

```diff
@@ -186,39 +186,42 @@
     "is_lang_es": """(user.profile.language() == "es")""",
     "is_not_redirect": """
      !( (user.profile.language() == "en" &&
-          (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasOptionStartsWith("C_SARCOMAS_OSTEOSARCOMA", "C_BRAIN_")
```

Comment:
changes in this are to keep in sync with what was updated in the SB task/patch SQL's, not really used by code/task